larpix/larpix.py

```diff
@@ -749,18 +757,33 @@ def parse_input(self, bytestream):
                     code = 'bytes:1='
                 byte_packets.append((Bits(code + str(metadata)),
                     Packet(current_stream[data_bytes])))
+                byte_packets[-1][1].cpu_timestamp = self.last_read_time
+                byte_packets[-1][1].read_id = self.read_counter
+                byte_packets[-1][1].bytestream_id = current_byte
+                        # store postion in bytestream of packet
                 current_stream = current_stream[packet_size:]
             else:
                 # Throw out everything between here and the next start byte.
                 # Note: start searching after byte 0 in case it's
                 # already a start byte
                 next_start_index = current_stream[1:].find(start_byte)
+                current_byte += next_start_index
                 current_stream = current_stream[1:][next_start_index:]
+
+        for chip in self.chips:
```

Comment:
The `Chip.read_data` list is updated every time a `parse_input` is called.
It contains the computer timestamp, read id, non-default config values, a log message (needs implementing), and a list of packets.

Reply:
Could you just save that stuff into `self.reads` instead of `self.read_data`?